debian/control

```diff
@@ -64,14 +64,16 @@ Package: ibverbs-providers
 Architecture: linux-any
 Multi-Arch: same
 Depends: ${misc:Depends}, ${shlibs:Depends}
-Provides: libcxgb3-1, libipathverbs1, libmlx4-1, libmlx5-1, libmthca1, libnes1
+Provides: libefa-1, libcxgb3-1, libipathverbs1, libmlx4-1, libmlx5-1, libmthca1, libnes1
 Replaces: libcxgb3-1 (<< 15),
+          libefa-1 (<< 15),
```

Comment:
Did a libefa-1 package exist? I do not see one in Debian or Ubuntu. Therefore the Replaces and Breaks on libefa-1 is not needed.

Reply:
No, thanks for this.